.github/workflows/linux.yml

```diff
@@ -6,14 +6,14 @@ on:
 
 jobs:
   Blint-GNU-Build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
This has to be built with the lowest linux and python versions to make it compatible with a broad range of os. Otherwise there will be glibc errors